### DISCUSSION TITLE
handbook/0.1/posts/20230117_adicionar_projeto_a_issue_via_actions/

### DISCUSSION THREAD

**Discussion by giscus** [transparencia-mg/handbook#35] [Announcements]
# handbook/0.1/posts/20230117_adicionar_projeto_a_issue_via_actions/

https://transparencia-mg.github.io/handbook/0.1/posts/20230117_adicionar_projeto_a_issue_via_actions/

**Comment by gabrielbdornas** [comment]
@Andrelamor, gostaria de sua opinião sobre o achado. Criei [este](https://github.com/transparencia-mg/handbook/issues/34) também para levarmos esta nova funcionalidade para todos os nosso repositórios.
